uniffi.toml

```diff
@@ -44,8 +44,26 @@ from_custom = "{}.toString()"
 [bindings.swift.custom_types.Timestamp]
 type_name = "Date"
 imports = ["Foundation"]
-into_custom = "{let df = DateFormatter();df.dateFormat = \"yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ\";return df.date(from: {})!}()"
-from_custom = "{let df = DateFormatter();df.dateFormat = \"yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ\";return df.string(from: {})}()"
+into_custom = """
+{
+    let s = {}
+    let df = ISO8601DateFormatter()
+    let fo = ISO8601DateFormatter.Options.withFractionalSeconds
+    df.formatOptions.insert(fo)
+    func t() -> Date? {df.date(from: s)}
```

Comment:
Not sure it improves readability to define this function, could just call `df.date(from: s)` directly?

Reply:
agree